### ISSUE TITLE
Wront fontification for Intepolation blocks

### ISSUE THREAD

**Issue by thomasf** [defunkt/coffee-mode#287]
I think this started to happen very recently but I'm not sure.

![image](https://cloud.githubusercontent.com/assets/330847/5979583/2d48566a-a8a9-11e4-89fa-736479920052.png)

```coffee
testMethod = (id) ->
  req = request.post("/api/methods/#{id}/test/")
  promise = reqPromise req
  promise.then (v) ->
    console.log v
    Stores.method.testResultAdded.dispatch(v)
  true
```

**Comment by syohex** [comment]
@thomasf Thanks for information. I fixed at #288. Would you check latest version ?

![coffee](https://cloud.githubusercontent.com/assets/554281/5985821/7cc9d614-a92b-11e4-9feb-ffcbb230a617.png)

**Comment by thomasf** [comment]
Works fine, thanks.
